jdk_8_maven/em/embedded/rest/blogapi/src/main/resources/blogapi.sql

```diff
@@ -186,7 +186,3 @@ CREATE TABLE `user_role` (
   CONSTRAINT `fk_security_user_id` FOREIGN KEY (`user_id`) REFERENCES `users` (`id`),
   CONSTRAINT `fk_security_role_id` FOREIGN KEY (`role_id`) REFERENCES `roles` (`id`)
 ) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8;
-
-LOCK TABLES `roles` WRITE;
-INSERT INTO `roles` VALUES (1,'ROLE_ADMIN'),(2,'ROLE_USER');
-UNLOCK TABLES;
```

Comment:
as this was part of the original SUT files, would be better to leave it as it was